server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -39,42 +40,69 @@ export default class ProbationPractitionerReferralsController {
     private readonly hmppsAuthService: HmppsAuthService,
     private readonly assessRisksAndNeedsService: AssessRisksAndNeedsService,
     private readonly draftsService: DraftsService,
-    private readonly referenceDataService: ReferenceDataService
+    private readonly referenceDataService: ReferenceDataService,
+    private readonly userDataService: UserDataService
   ) {
     this.deliusOfficeLocationFilter = new DeliusOfficeLocationFilter(referenceDataService)
   }
 
   async showOpenCases(req: Request, res: Response): Promise<void> {
     const pageSize = config.apis.interventionsService.dashboardPageSize.pp.openCases
-    await this.showDashboard(req, res, { concluded: false }, 'Open cases', pageSize)
+    await this.showDashboard(req, res, { concluded: false }, 'Open cases', 'ppOpenCases', pageSize)
   }
 
   async showUnassignedCases(req: Request, res: Response): Promise<void> {
     const pageSize = config.apis.interventionsService.dashboardPageSize.pp.unassignedCases
-    await this.showDashboard(req, res, { concluded: false, unassigned: true }, 'Unassigned cases', pageSize)
+    await this.showDashboard(
+      req,
+      res,
+      { concluded: false, unassigned: true },
+      'Unassigned cases',
+      'ppUnassignedCases',
+      pageSize
+    )
   }
 
   async showCompletedCases(req: Request, res: Response): Promise<void> {
     const pageSize = config.apis.interventionsService.dashboardPageSize.pp.completedCases
-    await this.showDashboard(req, res, { concluded: true, cancelled: false }, 'Completed cases', pageSize)
+    await this.showDashboard(
+      req,
+      res,
+      { concluded: true, cancelled: false },
+      'Completed cases',
+      'ppCompletedCases',
+      pageSize
+    )
   }
 
   async showCancelledCases(req: Request, res: Response): Promise<void> {
     const pageSize = config.apis.interventionsService.dashboardPageSize.pp.cancelledCases
-    await this.showDashboard(req, res, { cancelled: true }, 'Cancelled cases', pageSize)
+    await this.showDashboard(req, res, { cancelled: true }, 'Cancelled cases', 'ppCancelledCases', pageSize)
```

Comment:
💭 we have `type PPDashboardType = 'Open cases' | 'Unassigned cases' | 'Completed cases' | 'Cancelled cases'` -- I wonder if this new ID could be woven into that type, given they define the same concept?